ledger/catchupaccessor.go

```diff
@@ -330,12 +337,80 @@ func (c *CatchpointCatchupAccessorImpl) processStagingBalances(ctx context.Conte
 		}
 
 		normalizedAccountBalances, err = prepareNormalizedBalancesV6(balances.Balances, c.ledger.GenesisProto())
+		isNotFinalEntry = make([]bool, len(balances.Balances))
+		for i, balance := range balances.Balances {
+			isNotFinalEntry[i] = balance.IsNotFinalEntry
```

Comment:
why need to store? the look below always accesses the current value ie. `isNotFinalEntry[i]` while processing i'th entry

Reply:
The old version `catchpointFileBalancesChunkV5` won't have that field. This way it wont break with old catchpoint versions.